src/shared/components/case-editor/case-edit-submit/case-edit-submit.component.spec.ts

```diff
@@ -578,6 +585,9 @@ describe('CaseEditSubmitComponent', () => {
         declarations: [
           CaseEditSubmitComponent,
           IsCompoundPipe,
+          CcdPageFieldsPipe,
+          FieldsFilterPipe,
+          ReadFieldsFilterPipe,
```

Comment:
Why are they listed differently in these two sections? - Ccd... at bottom of first section, top of second